src/main/java/hudson/plugins/fitnesse/FitnesseExecutor.java

```diff
@@ -68,6 +71,11 @@ public boolean execute(Launcher launcher,FilePath workspace, Run<?,?> build) thr
 				throw (InterruptedException) t;
 			return false;
 		} finally {
+			try {
+				killTest(getFitnessePage(build, false));
```

Comment:
I think I see what you mean. I have changed the code so it will only be executed once the catch is executed.